temporal-serviceclient/src/main/java/io/temporal/serviceclient/RpcRetryOptions.java

```diff
@@ -93,6 +95,10 @@ public static final class Builder {
 
     private Duration maximumInterval;
 
+    // 0 is a valid value for maximumJitter, and yet, it is not the default value for maximumJitter.
```

Comment:
This comment doesn't make much sense to me.
What is "Not configured jitter"? For me it means "disabled jitter". Disabled jitter I believe is exactly maximumJitter = 0. Double.NaN is a "marker for default jitter".